### PR TITLE
Fix Makie issues

### LINKED ISSUE #989: Bugs due to Makie 0.20 update
Described in #934 :

- Fix 3D ContinuousSpace inspection
- Polygon plots are wonky/broken and run very slowly

### PR DESCRIPTION
**Pull request by Tortar**
Fixes #989

### CODE REVIEW

**Comment by Tortar** [comment]
I verified that now Flocking and Rabbit_Fox_Hawk work well

File: examples/rabbit_fox_hawk.jl [hunk 1]
```diff
@@ -151,7 +151,7 @@ function initialize_model(
     end
     for _ in 1:n_foxes
         pos = random_walkable(model, model.landfinder)
-        add_agent!(pos, Rabbit, model, v0, rand(abmrng(model), Δe_rabbit:2Δe_rabbit))
+        add_agent!(pos, Fox, model, v0, rand(abmrng(model), Δe_rabbit:2Δe_rabbit))
```
Comment (Datseris) [inline examples/rabbit_fox_hawk.jl:154]: I'm confused, we add a fox but we have a `Δe_rabbit`?

Comment (Tortar) [reply examples/rabbit_fox_hawk.jl:154]: I was confused to, but

`    Δe_rabbit = 30,  ## energy gained from eating one rabbit`

Comment (Tortar) [reply examples/rabbit_fox_hawk.jl:154]: It doesn't make much sense anyway though as an initialization

Comment (Tortar) [reply examples/rabbit_fox_hawk.jl:154]: or maybe yes, it's like they start with some rabbit already eaten :D

**Review by Datseris — APPROVED** [review]

**Pull request merged**